### ISSUE TITLE
no CLI, and doesnt work with Python

### ISSUE THREAD

**Issue by pirogieses** [ddinan/cstrips-downloader#4]
Hello! I made a github account just to write this. I'm an ex Bitstrips user and I wanted to download my comics, but I don't have a CLI, and even after downloading Python I couldnt get the downloader to work. Where the step-by-step guide tells me to introduce my alphanumerical code, I see nowhere to introduce it - almost as if the script is broken. I see no [USER] where I should introduce my code!

I am not a programmer but my dad is, and when I asked for his help, he couldn't get it to work, either. If you could help me with this, I would appreciate it greatly! If you want to contact me directly (I dont know how communication works in here) you can contact my instagram (pirogieses) or my gmail (gromow2012@gmail.com). 

Thank you in advance, and I so greatly appreciate your work! My friend got their comics back through this downloader, so I KNOW it works :)

**Comment by ddinan** [comment]
Could you please tell me which OS and version you are using? Most should include a terminal/command prompt which you can typically find by searching "cmd" into the search bar. Are you also using the compiled (.exe version) or custom-built (direct Python version)?

**Comment by pirogieses** [comment]
My OS is Windows 7 (very old, I know) and I tried both, and they both failed. I did use the CMD prompt to no avail. I will try again today, considering now I have more stable wi-fi, but from what I remember both versions with and without Python failed due to the absense of a CLI (an error message appeared saying as such).

Thank you for your reply! And I hope I can figure out where the problem is :(
